fastapi/utils.py

```diff
@@ -21,6 +21,16 @@
 def is_body_allowed_for_status_code(status_code: Union[int, str, None]) -> bool:
     if status_code is None:
         return True
+    # Ref: https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#patterned-fields-1
```

Comment:
FastAPI supports only 3.0.2, not 3.1.0.

Both versions are compliant with this change.

Reply:
Right!